pkg/config/config.go

```diff
@@ -16,12 +16,79 @@
 
 package config
 
-import "fmt"
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
 
 const (
-	filenameSuffix = "_test_configuration"
+	configurationFilePathEnvironmentVariableKey = "TEST_CONFIGURATION_PATH"
```

Comment:
The more I think about it (and I realize this is from my submission originally), we should prefix with `TNF_`.